src/app/home/components/DataList/Clusters/ClusterItem.tsx

```diff
@@ -45,10 +52,23 @@ const ClusterItem = ({ cluster, clusterIndex, isLoading, migMeta, ...props }) =>
             <DataListCell key="actions" width={2}>
               <Flex justifyContent="flex-end">
                 <Box mx={1}>
-                  <Button variant="secondary">Edit</Button>
+                  <Button onClick={toggleOpen} variant="secondary">Edit</Button>
+                  <AddClusterModal isOpen={isOpen} onHandleClose={toggleOpen} name={clusterName}
+                   url={clusterUrl} token={clusterSvcToken} mode="update"
+                   />
                 </Box>
                 <Box mx={1}>
-                  <Button variant="danger">Remove</Button>
+                    <Button onClick={() => {
+                      // TODO:  Not sure best way to ensure that user wants to really delete this item.
+                      if (confirm('Are you sure?')) {
```

Comment:
This is kind of ugly popping up an 'alert'.

Assuming there is a better approach to making this cleaner.

Reply:
I think an alert popup is pretty standard although I am not seeing a PF4 component for one unfortunately. That may just mean we need to just style the popup to look like pf4. 

something like this https://react.semantic-ui.com/addons/confirm/#types-confirm

Reply:
Maybe we can use some PF4 modal?

Reply:
@jwmatthews added a pf4 modal component for this. going to merge in to your branch .